CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- ⚠️ Auto IP assignment for `SentryUser` is now guarded by `sendDefaultPii` ([#2726](https://github.com/getsentry/sentry-dart/pull/2726))
```

Comment:
@kahest I added it to the top without a category for most visibility, wdyt?

Reply:
placement makes sense 👍 please also add a header called "Behavioral changes"